### ISSUE TITLE
Add "Sleep" mode for ACs

### ISSUE THREAD

**Issue by digitalheartxs** [gvigroux/hon#151]
Would be great to add the possibility to turn off AC's display for the night by home assistant.

**Comment by gvigroux** [comment]
There is an action you can trigger anytime:
hOn: Set climate screen display

Can you please test ?

**Comment by sirfalo** [comment]
Hi

Screen off is different function. Today I upgraded (I had version 7) and Night Mode was working normally, now it dissapeared. 
It differs from "screen off" because night mode increases temperatur each 2 or 3 hours by 1°C.

I found action: hOn: Set climate sleep mode
I confirm it works same way, but previous way was better I have to say :)
I could just use this as button without any additional added scripts or whatever.

@gvigroux Awesome work that you maintain this integration :)
